armory/eval_scripts/poisoning_attack_backdoor.py

```diff
@@ -65,6 +65,7 @@ def evaluate_classifier(config_path: str) -> None:
     classifier_fn = getattr(classifier_module, model_config["name"])
     preprocessing_fn = getattr(classifier_module, "preprocessing_fn")
 
+    logger.info(f"Loading dataset {config['dataset']['name']}.")
```

Comment:
Mind making all of these `...` to specify to users that it could take a few moments.